result_quality.md

```diff
@@ -0,0 +1,56 @@
+# Determining Result Quality
+
+Pelias tries very hard to always return good results, and generally does a pretty good job. If using the autocomplete endpoint or otherwise showing results directly to humans, judging result quality is generally easy (if subjective).
```

Comment:
this line:

> If using the autocomplete endpoint or otherwise showing results directly to humans, judging result quality is generally easy (if subjective).

.. is quite long and hard to follow the meaning.

I read it as "Judging result quality is easy. This is especially true when using autocomplete, or alternatively when showing results directly to an end user.

.. but even written like that it doesn't really say much, I would suggest removing it, as eyeball congestion this early in the document might put people off?